abi.html

```diff
@@ -5628,9 +5628,9 @@ <h4><a href="#expressions">5.1.6 Expressions</a></h4>
 syntactic expression tree, with parentheses omitted.  (Parentheses
 may be ignored because they are implicit in the prefix representation
 and typically do not affect semantics.  However, a parenthesized
-<code>&lt;<a href="#mangle.unresolved-name">unresolved-name</a>&gt;</code>
-must be mangled differently because the parentheses act to suppress
-argument-dependent lookup.)  Unless explicitly stated otherwise, the
+<code>&lt;<a href="#mangle.base-unresolved-name">base-unresolved-name</a>&gt;</code>
+must be mangled differently when argument-dependent lookup would occur
+without the parentheses.)  Unless explicitly stated otherwise, the
```

Comment:
```suggestion
are used to suppress argument-dependent lookup, the call expression may
need to be mangled differently.  Unless explicitly stated otherwise, the
```